### DISCUSSION TITLE
www

### DISCUSSION THREAD

**Discussion by MuhammadWachidin** [filamentphp/filament#14833] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

![Screenshot 2024-11-18 230923](https://github.com/user-attachments/assets/43407962-aea6-4271-ac3f-b21bee20f933)
NOT SHOW TGL DIAMBIL :(

**Comment by MuhammadWachidin** [comment]
> ### Package
> Table builder
> 
> ### Package Version
> v3.0.0
> 
> ### How can we help you?
> ![Screenshot 2024-11-18 230923](https://private-user-images.githubusercontent.com/114646249/387297607-43407962-aea6-4271-ac3f-b21bee20f933.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE5NDY0NjEsIm5iZiI6MTczMTk0NjE2MSwicGF0aCI6Ii8xMTQ2NDYyNDkvMzg3Mjk3NjA3LTQzNDA3OTYyLWFlYTYtNDI3MS1hYzNmLWIyMWJlZTIwZjkzMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExOFQxNjA5MjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YzMyZWY3MzEzMzBiMjEwYzg0OTM1ZWFhZTViNzZkYWE4N2E4Y2M1MDAzNGRmMDdlYWE5YWE4ZDQxNjk1NjMyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.SfAxDygDhONYQBa1pL9OYxHjbll_KGo89SviTxGxp4g) NOT SHOW TGL AMBIL :(

**Comment by miiiwdy** [comment]
mana codenya?
